test/unit/app/services/bulk_republisher_test.rb

```diff
@@ -0,0 +1,22 @@
+require "test_helper"
+
+class BulkRepublisherTest < ActiveSupport::TestCase
+  extend Minitest::Spec::DSL
+
+  describe "#republish_all_organisation_about_us_pages" do
+    test "queues Organisation 'About Us' pages for republishing" do
+      queue_sequence = sequence("queue")
+
+      2.times do
+        about_us_page = create(:about_corporate_information_page)
+
+        PublishingApiDocumentRepublishingWorker
+          .expects(:perform_async_in_queue)
+          .with("bulk_republishing", about_us_page.id, true)
+          .in_sequence(queue_sequence)
+      end
```

Comment:
If there's a preference for doing setup before (pre-)assertions, we could do this, but the current approach feels easier to read

```rb
      Array.new(2).map { create(:about_corporate_information_page) }.each do |about_us_page|
        PublishingApiDocumentRepublishingWorker
          .expects(:perform_async_in_queue)
          .with("bulk_republishing", about_us_page.id, true)
          .in_sequence(queue_sequence)
      end
```